lib/content-services/src/lib/tag/tags-creator/tags-creator.component.ts

```diff
@@ -369,6 +372,13 @@ export class TagsCreatorComponent implements OnInit, OnDestroy {
             : null;
     }
 
+    private validateSpecialCharacters(tagNameControl: FormControl<string>): TagNameControlErrors | null {
+        const specialSymbolsRegex: RegExp = /[':"\\|<>\/?]/;
```

Comment:
no need to set type, typescript sees that you use RegExp as you wrap it in // so it sets type automatically: 
<img width="1009" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/4de16203-033f-46a3-bee5-3cc685068fac">